src/svc-cloud-data-sources.yml

```diff
@@ -0,0 +1,24 @@
+openapi: 3.1.0
+info:
+  title: Cloud Data Sources Service
+  version: '1.0'
+servers:
+  - url: /
+paths:
+  '/orgs/{orgID}/datasources':
```

Comment:
I believe we are wanting to move away from having the orgid in the route if possible. We're currently deriving the orgId from the token/session but i think there is a recent service that is still accepting orgID as a query parameter too.